docs/3.api/3.utils/once.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "once"
+description: "Run a given function or block of code once SSR or CSR."
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/composables/error.ts
+    size: xs
+---
+
+## Purpose
+
+The `once` function is designed to execute a given function or block of code only a single time in a specific scenario:
+- During server-side rendering but not during hydration
+- On client-side navigation
+
+This is useful for code that should only be executed once, such as logging an event or setting up a global state.
+
+## Usage
+
+```vue [app.vue]
+<script setup>
+const websiteConfig = useState('config')
+
+await once(async () => {
+  console.log('This will only be logged once')
+  websiteConfig.value = await $fetch('https://my-cms.com/api/website-config')
+})
+</script>
+```
+
+::callout{to="/docs/getting-started/state-management#usage-with-pinia"}
+`once` is useful in combination with the [Pinia module](/modules/pinia) to call store actions.
+::
+
+:read-more{to="/docs/getting-started/state-management"}
+
+::callout{color="info" icon="i-ph-warning-duotone"}
+Note that `once` won't return anything, you should use [`useAsyncData`](/docs/api/composables/use-async-data) or [`useFetch`](/docs/api/composables/use-fetch) if you want to do data-fetching during SSR.
+::
+
```

Comment:
```suggestion

::callout
`once` is a composable meant to be called directly in a setup function, plugin, or route middleware, because it needs to add data to the Nuxt payload to avoid re-calling the function on the client when the page hydrates.
::

```